### PR TITLE
chore(dag): fix dag block issue

### PR DESCRIPTION
**Pull request by MatusKysel**

### CODE REVIEW

File: src/storage/db_storage.cpp [hunk 1]
```diff
@@ -219,6 +219,18 @@ std::shared_ptr<DagBlock> DbStorage::getDagBlock(blk_hash_t const& hash) {
   return nullptr;
 }
 
+bool DbStorage::checkDagBlock(blk_hash_t const& hash) {
```
Comment (mfrankovi) [inline src/storage/db_storage.cpp:222]: We have transactionInDb and pbftBlockInDb methods, maybe dagBlockInDb would be a better name than checkDagBlock

File: src/node/full_node.cpp [hunk 2]
```diff
@@ -221,7 +221,7 @@ void FullNode::start() {
         received_blocks_++;
       }
 
-      if (dag_blk_mgr_->pivotAndTipsAvailable(blk)) {
+      if (dag_mgr_->pivotAndTipsAvailable(blk)) {
```
Comment (mfrankovi) [inline src/node/full_node.cpp:224]: Maybe put check inside of the addDagBlock and have it return a bool. But this is fine as well

**Review by mfrankovi — APPROVED** [review]

File: src/node/full_node.cpp (hunk 2 above)
Comment (MatusKysel) [reply src/node/full_node.cpp:224]: addDagBlock is used also elsewhere so I think I can not do it

File: src/storage/db_storage.cpp (hunk 1 above)
Comment (MatusKysel) [reply src/storage/db_storage.cpp:222]: good point I will fix it

**Review by justinsnapp — APPROVED** [review]

**Pull request merged**